docs/cosmos-hub/upgrade-node.md

```diff
@@ -0,0 +1,99 @@
+# Upgrade Your Node
+
+::: warning
+The detailed procedure to upgrade a mainnet node from `cosmoshub-1` to `cosmoshub-2` can be found [here](https://gist.github.com/alexanderbez/5e87886221eb304b9e85ad4b167c99c8). 
+:::
+
+This document describes the upgrade procedure of a `gaiad` full-node from a version <current_version> to a version <new_version>.
+
+## Software Upgrade
+
+First, stop your instance of `gaiad`. Next, upgrade the software:
+
+```bash
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+git fetch --all && git checkout <new_version>
+make tools install
+```
+
+::: tip
+*NOTE*: If you have issues at this step, please check that you have the latest stable version of GO installed.
+:::
+
+See the [testnet repo](https://github.com/cosmos/testnets) for details on which version is needed for which public testnet, and the [SDK release page](https://github.com/cosmos/cosmos-sdk/releases) for details on each release.
+
+Your full node has been cleanly upgraded!
+
+## Fetch new genesis 
+
+:::warning 
+If the version <new_version> you are upgrading to is breaking from the previous one, you will have to restart your chain. If it is not breaking, you can skip to [Restart](#restart)
+:::
+
+The first step is to remove your current genesis:
+
+```bash
+rm $HOME/.gaiad/config/addrbook.json $HOME/.gaiad/config/genesis.json
```

Comment:
I don't think users will want to remove their addrbook?

Reply:
oh this should not be there, good catch